packages/components/src/components/scroller/scroller.examples.md

```diff
@@ -175,10 +181,39 @@ export default class Code extends Vue {
   </p-scroller>
 </div>`;
   }
+  scrollbar = `<div style="max-width: 600px">
```

Comment:
Corresponding closing tag is missing which leads to build error in stackblitz.
<img width="447" alt="Bildschirm­foto 2023-03-22 um 07 26 03" src="https://user-images.githubusercontent.com/4380295/226819719-70fbf5e4-2c24-4896-94e3-1dc9abf6209b.png">

Reply:
Fixed it